Python/pywarpx/Bucket.py

```diff
@@ -68,3 +68,5 @@ def attrlist(self):
             result += [attrstring]
         return result
 
+    def __setstate__(self, state):
+        pass
```

Comment:
Without it `runinfo` doesn't work due to an issue with recursion depth - we had this issue before with Warp as well and resolved it through the same approach. I guess it depends on what you want to pickle as to whether it will break. I believe this will break some parts of `mwxrun.simulation` if we tried to pickle it but until we find something broken by it, I'd say we should keep it in since it allows `FluxDiagFromFile` to work (specifically allows `runinfo` to be loaded). I will add the comment explanation though.